cloudinstall/install.py

```diff
@@ -122,10 +126,8 @@ def do_install(self, install_type):
                     self.MultiInstallNewMaas(
                         self.loop, self.ui, self.config).run()
             else:
-                self.ui.set_openstack_rel("Icehouse (2014.1.3)")
```

Comment:
@mikemccracken, @jamesbeedy yea this was put within the callback functions and is not called when headless is True.

NACK on this change